app/controllers/admin/application_controller.rb

```diff
@@ -9,9 +9,14 @@ class ApplicationController < Administrate::ApplicationController
     before_action :authenticate_educator!, :authenticate_admin
 
     def authenticate_admin
-      redirect_to(new_educator_session_path) unless current_educator && current_educator.can_set_districtwide_access?
+      redirect_to(not_authorized_path) unless current_educator && current_educator.can_set_districtwide_access?
```

Comment:
This was not-quite-right previously, and it came up in testing this PR so I fixed it up.